packages/react/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+module.exports = () => {
+  const { WITH_REPORT } = process.env;
+
+  return {
+    mode: 'production',
+    entry: './src/index.ts',
+    output: {
+      path: path.resolve(__dirname, 'dist'),
+      filename: 'logto.react.min.js',
+      library: {
+        type: 'umd',
+        name: 'logto',
```

Comment:
should be `logto.react` or `logto-react`?

Reply:
Follow the [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1644208989260219) on Slack

<img width="893" alt="image" src="https://user-images.githubusercontent.com/10594507/155939733-90af24c7-ae57-444d-a9e2-b145bf8c3246.png">

Reply:
Removed `Webpack`. Do not support `package` for now and add it when needed.